src/main/java/org/opentripplanner/gtfs/mapping/GTFSToOtpTransitServiceMapper.java

```diff
@@ -133,33 +156,46 @@ private void mapAndAddTransfersToBuilder() {
         builder.getTransfers().addAll(transferMapper.map(data.getAllTransfers()));
     }
 
-
-    private void mapGtfsStopsToOtpTypes(GtfsRelationalDao data) {
+    private void mapGtfsStopsToOtpTypes(GtfsRelationalDao data, TranslationHelper translationHelper) {
         StopToParentStationLinker stopToParentStationLinker = new StopToParentStationLinker(issueStore);
         for (org.onebusaway.gtfs.model.Stop it : data.getAllStops()) {
             if(it.getLocationType() == org.onebusaway.gtfs.model.Stop.LOCATION_TYPE_STOP) {
-                Stop stop = stopMapper.map(it);
+                Stop stop = stopMapper.map(it,
+                        translationHelper.getTranslation(TABLE_STOPS, STOP_NAME, it.getId().getId(),
```

Comment:
I think we should just pass in the translation helper into the stop mapper, and do the fetching in there

Reply:
Done.

Reply:
I think it would be nicer, if it was passed in as a constructor parameter, not to the individual map calls.

Reply:
I think it would be nicer, if it was passed in as a constructor parameter, not to the individual map calls. It would simplify the code somewhat